src/components/pet/view/FormModal.vue

```diff
@@ -0,0 +1,264 @@
+<template>
+  <div>
+    <b-modal
+      id="modal-prevent-closing"
+      ref="modal"
+      v-bind:title="$t('pet.new_pet')"
+      size="lg"
+      @show="resetModal"
+      @hidden="resetModal"
+      @ok="handleOk"
+    >
+      <form ref="form" @submit.stop.prevent="handleSubmit">
+        <div class="text-center"><b-avatar v-bind:src="pet.avatar" size="6rem"></b-avatar></div>
+        <span class="text-error" v-if="errors && errors.code == 500">
```

Comment:
cái này là http status code nhỉ, nếu thế cần định nghĩa constants riêng
cơ mà pet not exist là 500 à :v

Reply:
phần này t quên chưa xóa c ạ